test/unit/webauthn_verification_test.rb

```diff
@@ -50,4 +50,23 @@ class WebauthnVerificationTest < ActiveSupport::TestCase
       end
     end
   end
+
+  context "#generate_otp" do
+    setup do
+      @webauthn_verification = create(:webauthn_verification, otp: nil, otp_expires_at: nil)
+      @generated_time = Time.utc(2023, 1, 1, 0, 0, 0)
+      travel_to @generated_time do
+        @webauthn_verification.generate_otp
+      end
+      @webauthn_verification.reload
+    end
+
+    should "create a token that is 20 characters long" do
+      assert_equal 20, @webauthn_verification.otp.length
+    end
```

Comment:
IIRC we're using 16 characters for the path_token. Should we use the same here?

Reply:
Good point, I was basing the number off of the number set in the factory which was 20. I like the consistency of 16 so I changed it to that ✅

Reply:
Aha! I thought I'd changed that in #3328, but I think it got lost in the general rebase shuffle.